src/reducers/paths.reducer.js

```diff
@@ -41,12 +43,16 @@ export default (state: State = initialState, action: Action) => {
 //
 //
 // Helpers
-export const getDefaultParentPath = () =>
+export const getDefaultParentPath = () => {
   // Noticing some weird quirks when I try to use a dev project on the compiled
   // "production" app, so separating their home paths should help.
-  process.env.NODE_ENV === 'development'
-    ? `${os.homedir()}/guppy-projects-dev`
-    : `${os.homedir()}/guppy-projects`;
+
+  const homedir = isWin() ? getWindowsHomeDir() : os.homedir();
```

Comment:
I wonder if it make sense to add a new method in `platform.services` like this?

```js
export const getWindowsHomeDir = () => {
  // unchanged
}

export const getHomeDir = () => (
  isWin() ? getWindowsHomeDir() : os.homedir()
);
```

That way, in this file, we can just import and use `getHomeDir`, so that `paths.reducer` doesn't have to know about platform-specific quirks.

I don't feel that strongly about it, so feel free to leave as-is if you think that extra layer of abstraction isn't useful.

Reply:
this result of this function should be cached as well